deepcell_tracking/tracking.py

```diff
@@ -1100,5 +1037,105 @@ def _remove_false_pos(self, lineage, tracked, FP_info):
 
         return lineage, tracked
 
+    # def _get_input_pairs(self, frame):
+    #     """Get all input pairs, inputs, and invalid pairs.
+
+    #     Args:
+    #         frame (int): Returns input pairs for only the given frame.
+
```

Comment:
Just delete this, no reason to keep around. We can just review one of the old releases to compare.